templates/web/pages/product_edit/product_edit_form_display.tt.js

```diff
@@ -34,3 +34,67 @@
 		\$('#nutrition_data_div').show();
 	}
 });
+
+function show_warning(should_show, nutirent_id, warning_message){
+	if(should_show) {
+		\$('#nutriment_'+nutirent_id).css("background-color", "rgb(255 237 235)");
+		\$('#nutriment_question_mark_'+nutirent_id).css("display", "inline-table");
+		\$('#nutriment_sugars_warning_'+nutirent_id).text(warning_message);
+	}else {
+		\$('#nutriment_'+nutirent_id).css("background-color", "white");
+		\$('#nutriment_question_mark_'+nutirent_id).css("display", "none");
+	}
+}
+
+var sugars_value;
+var carbohydrates_value;
+var saturated_fats_value;
+var fat_value;
+
+var required_nutrients_id = ['energy-kj', 'energy-kcal', 'fat', 'saturated-fat', 'sugars', 'carbohydrates', 'fiber', 'proteins', 'salt', 'sodium', 'alcohol'];
+
+required_nutrients_id.forEach(nutirent_id => {
```

Comment:
@stephanegigandet, using the ordinary for loop is tricky.  Only the last nutrient, "alcohol" is accessed the colour but the `on-input` event exists on other elements 

Is this okay?

Reply:
It looks good this way, thank you! :)

Could you just replace all the "nutirent" by "nutrient" ? :)

Reply:
thanks @stephanegigandet 
That slipped my eyes